src/server/schemas/nft/index.ts

```diff
@@ -222,11 +225,11 @@ export const signature721OutputSchema = Type.Object({
         "If you want the user to pay for minting the tokens, you can specify the price per token. Defaults to 0.",
     }),
   ),
-  mintStartTime: Type.Number({
+  mintStartTime: Type.Integer({
```

Comment:
is it intended that some of these have minimum: 0 and some dont?

Reply:
I didn't put minimum on response bodies (output), only request bodies (input). Also some I wasn't super confident on, so I left them untouched.